### ISSUE TITLE
Copying a source does not put transform information on the clipboard

### ISSUE THREAD

**Issue by Lego6245** [obsproject/obs-studio#6035]
### Operating System Info

Windows 10

### OBS Studio Version

27.2.1

### OBS Studio Log URL

https://obsproject.com/logs/QkIWxAd4tC9jYmd8

### Expected Behavior

When copying sources (right click -> copy or ctrl+c) in versions prior to 27.2.1, you could right click -> transform -> paste transform and apply the transform of the copied source to the target.

### Current Behavior

When copying sources (right click -> copy or ctrl+c) in versions prior to 27.2.1, it does not put transform related information on the clipboard. Right click -> transform -> paste transform applies the last transform copied via Right click -> transform -> copy transform

### Steps to Reproduce

1. Have a scene with 2 sources with different transforms (positions, sizes, etc).
2. Ctrl + C or Right click -> copy one source.
3. Right click -> transform -> paste transform on the second source.
...

### Anything else we should know?

This may end up being a wontfix, but I was relying heavily on the prior behavior so the loss of it feels like an unintentional bug/side effect of the recent fix for copying sources between scenes maintaining their transforms (which is a great fix).

**Comment by RytoEX** [comment]
I believe this is due to #5340 and is probably by design.  Will let others weigh in.

**Comment by gxalpha** [comment]
I agree that this is by design. There was discussions about a potential shortcut for Copy/Paste Transform, not sure where that has gone.

**Comment by Lego6245** [comment]
Understandable if this ends up being by design.

Would love to see that shortcut added as I am often copy/pasting transforms with my current scene workflow.
